pep-0581.rst

```diff
@@ -0,0 +1,310 @@
+PEP: 581
+Title: Using GitHub Issues for CPython
+Version: $Revision$
+Last-Modified: $Date$
+Author: Mariatta Wijaya <mariatta@python.org>
+Status: Draft
+Type: Process
+Content-Type: text/x-rst
+Created: 20-Jun-2018
+
+
+Abstract
+========
+
+This PEP outlines the steps required to migrate Python's issue tracker
+from Roundup to using GitHub Issues.
+
+
+Rationale
+=========
+
+CPython's development moved to GitHub on February 2017. All other projects within
+The PSF's organization are hosted on GitHub and are using GitHub issues.
+CPython is still using Roundup as the issue tracker in bugs.python.org (bpo).
+
+Why GitHub
+----------
+
+It has a lot of nice features readily available out of the box, that are not
+currently available in Roundup / bpo.
+
+- APIs we can use to build integrations and automations.
+
+- Ability to attach screenshots and files in pull requests and issues.
+
+- Administrators and core developers can edit issues comments and pull requests.
+
+- Ability to reply to issues and pull request conversations via email.
+
+- It supports two factor authentication.
+
+- It supports markdown and emoji.
+
+- Core developers don't have to maintain it, giving us more time to focus on
+  the development of Python.
+
+- We can automatically close issues when a PR has been merged [#close-github-issue]_.
+
+- With more than 28 million users, an open source contributor is more likely
+  to already have an account, and familiar with GitHub interface, making it
+  easier to start contributing.
+
+- Email notifications contain metadata [#email-meta]_, integrated with GMail, and
+  allows you to systematically filter emails.
+
+- Email address can remain hidden by choice.
+
+
+Issues with bpo / Roundup
+-------------------------
+
+- It is not actively maintained.
+
+- Less than five people maintain bpo. Some of them are core developers.
+
+- It is in mercurial. Without any CI available, it puts heavy burden on existing
+  (few) maintainers in terms of reviewing, testing, and applying patches.
```

Comment:
It would help to count them. I counted 2, but it's better to ask Ezio for a more accurate answer. Ezio recently wrote: https://mail.python.org/pipermail/python-committers/2018-June/005515.html

In the mercurial logs, I see commits from 3 developers: Ezio, Berker, Maciej.